.conform.yaml

```diff
@@ -0,0 +1,26 @@
+policies:
```

Comment:
This should probably be submitted separately I think. it's not related to DNS configuration :)